services/QuillLMS/app/workers/amplify_report_activity_worker.rb

```diff
@@ -0,0 +1,25 @@
+class AmplifyReportActivityWorker
+  include Sidekiq::Worker
+
+  def perform(amplify_session_id, activity_name, description, score, activity_url, results_url)
+    @payload = generate_payload(amplify_session_id, activity_name, description, score, activity_url, results_url)
+    submit_payload_to_amplify(@payload)
```

Comment:
you shouldn't need to call this with the @payload argument since it's an instance variable